### ISSUE TITLE
Here is my hypothesis while training dataset.

### ISSUE THREAD

**Issue by Taekyoon** [NVIDIA/waveglow#181]
While I was training with my dataset, here are some hypotheses that model can train very stable.
I want to double-check with you guys because I want to gather other opinions from these hypotheses. 

- First, audio should be clear and constant with background and voice tone and pitches
- Second, try to remove short audio data which can make pads
- Third, try to ignore std of sampled audio while training
- Last, if there's underfitting happens, try to reduce learning rate first and reduce segment length if the first method doesn't work.

These hypotheses just came from opinions from the issues and my experiences.
Does anyone agree with this?

Please let me know if my hypotheses are not right.
Thanks

**Comment by NoahDrisort** [comment]
Thank you very much, can you share more detail about your parameters

**Comment by Taekyoon** [comment]
@NoahDrisort There's not much difference between default setups from `config.json`. I changed only learning rate and segment lengths and fp16_run.

Try make half values from hyper-params.

**Comment by MariaMsu** [comment]
Hi. Could you please explain what did you mean by
> Third, try to ignore std of sampled audio while training  

?

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
